platform/tron/api.go

```diff
@@ -274,11 +274,18 @@ func (p *Platform) GetDelegations(address string) (blockatlas.DelegationsPage, e
 }
 
 func (p *Platform) GetBalance(address string) (string, error) {
-	//TODO: Implement fetching balance for Tron
+	account, err := p.client.GetAccount(address)
+	if err != nil {
+		return "0", err
+	}
+
+	for _, data := range account.Data {
```

Comment:
if we have more them one `account.Data`?

Reply:
It is the same stuff as in Cosmos: https://github.com/trustwallet/blockatlas/blob/daec2de4faeac2e9815ec51d65a60aa12c408d6b/platform/cosmos/api.go#L133

I don't know why they enclose balances in the arrays perhaps for tokens transferring.